deliverables/README.md

```diff
@@ -8,61 +10,122 @@ To answer questions, please just put the answer on the next line after the quest
 
 Example:  
 
-**What is your name?**
-Sir Student of Ruby
+**What is your name?**  
+Jack Policar
 
-**What is your quest?**
-I seek the Rails  
+**What is your quest?**  To learn the basics of Ruby on Rails
+ 
 
-**What is your favorite color?**
-~~Green~~ edit:  No, blue.  Blue!  
+**What is your favorite color?**  Today, still green. 
+  
 
 For projects on Code Academy, or other external exercises, please just paste a link into your deliverables file where we can see the result.  
 
 ### Section 1: The Front End
 #### Required - Answer these questions in the deliverables file, and complete the following. 
-What is the role of HTML in a web page?  
-What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+**What is the role of HTML in a web page?**  <br/>--To create the skeleton of the website.   
+
+
+**What is the role of CSS in a web page?**  <br>To create the styling on the page. 
+
+
+**What is the role of Javascript in a web page?**  <br/>Java script is a and object scripting language it is used with html. It does not however incorporate or abide by any html tags or rules. 
+
+
+JavaScript, an object scripting language is used in web pages along with mark up language HTML. But JavaScript does not incorporate or abide by any HTML tags or rules.
 
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### meta
-Why do we separate HTML and CSS?  
+**Why do we separate HTML and CSS?** <br/>So you can easily organize and seperate what you are working on and keep track of the changes you have made once you get into larger files and programs.  
 
 #### HTML
-What are classes and IDs (and how are they different)?  
-What are elements?  
-What are tags?  
-What are attributes?  
-What are forms?  
-What is a div?  
-
-What are selectors?  
-What are properties?  
-What are values?  
-How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
-What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+**What are classes and IDs (and how are they different)?**   <br/> class selectors match an element based on the contents of the element's class attribute. Id selectors match an element based on the contents of that element's ID attribute
+
+
+**What are elements?**  <br/>Html element is an individual component of and HTML doc or webpage. 
+
+
+**What are tags?**  <br/>It is a code that describes how a web page is formatted. 
+
+
+**What are attributes?**  <br/>An attribute is something on a page you can set such as a font. 
```

Comment:
An attribute is a key-value pair inside an html tag.  

example: `<a href="some_url.com">Click here to go to SomeUrl</a>`  

Here "href" is an attribute.